installer/config.yaml

```diff
@@ -17,7 +17,7 @@ rhtapCLI:
       enabled: &rhdhEnabled true
       namespace: *installerNamespace
       properties:
-        catalogURL: https://github.com/redhat-appstudio/tssc-sample-templates/blob/release/all.yaml
+        catalogURL: https://github.com/redhat-appstudio/tssc-sample-templates/blob/v1.3.0/all.yaml
```

Comment:
I wonder if for the main branch of rhtap-cli, this should use the main branch of tssc-sample-templates?

Presumably, this will have to change regardless for 1.4?

Reply:
I don't want to use the main branch by default because in the past we had unexpected failure when the main branch was in a bad state.